packages/govern-subgraph/schema.graphql

```diff
@@ -34,25 +35,25 @@ type Action @entity {
   to: Bytes!
   value: BigInt!
   data: Bytes!
-  item: Item!
-  execution: Execution
+  execution: Execution!
+  queueItem: QueueItem!
 }
 
 type GovernQueue @entity {
   id: ID!
   address: Bytes!
   config: Config
+  queued: [QueueItem!]
+  challenges: [Challenge!]
+  vetos: [Veto!]
+  executions: [Execution!] @derivedFrom(field: "queue")
```

Comment:
Hmmm, I think we need to further see what's the relation between a `QueueItem` and an execution. For me, a `QueueItem` becomes an `Execution` after the executor has called its `exec` function. Mostly thinking about duplicated entities.

Reply:
You are right, I think there are room for optimize the number of entities.